draft-ietf-masque-h3-datagram.md

```diff
@@ -583,6 +404,32 @@ loss; this can misrepresent the true path properties, defeating methods such a
 DPLPMTUD.
 
 
+## Registering Datagram Contexts with Capsules {#register-capsules}
+
+Methods or protocols that support multiple datagram payload formats or
```

Comment:
maybe qualify "methods or protocols" here?

Reply:
Qualified in the long form explanation